brave-clean-slate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
The Windows version has a flag `--delete_profiles`. Instead of adding the new file `brave-clean-slate.sh`, could you add a similar flag to `uninstall-brave.sh`?